docs/components/table.md

```diff
@@ -107,6 +107,131 @@ Set this to `true` to disable ablity to sort.
 **Experimental** - set this prop to `true` to enable client side sorting if using a fetcher that returns unpaginatinated data.
 This functionality may be flaky.
 
+### sortHandlerFn
+
+Use a custom sort handler function to handle sorting table data for specific columns
+
+::: tip Notes
+
+1. In order for a column to use the custom sort handler function, `useSortHandlerFn` must be set to `true` in the [headers](#headers) object
+
+2. Sort handler functions can take four params:
+
+- `key`: the key of the column to be sorted
+- `prevKey`: the key of the column previously sorted
+- `sortColumnOrder`: the order in which to sort the column (`asc` or `desc`)
+- `data`: the data returned from the fetcher function response
+:::
+
+#### sortHandlerFn Example
+
+Here the `last_seen` column is set to use the custom sort handler function via the `useSortHandlerFn` property set in the table header object. The function passed into the `sortHandlerFn` prop sorts and returns the table data. The other columns use the default built-in client side sort function because the `useSortHandlerFn` property is not set in the header objects.
+
+<KTable
+  :fetcher="sortHandlerFnFetcher"
+  :headers="sortHandlerFnHeaders"
+  :sortHandlerFn="sortHandlerFn"
+  enable-client-sort
+/>
+
+```vue
+<template>
+  <KTable
+    :fetcher="fetcher"
+    :headers="headers"
+    :sortHandlerFn="sortHandlerFn"
+    enable-client-sort
+  />
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      headers: [
+        { label: 'Host', key: 'hostname', sortable: true },
+        { label: 'Version', key: 'version', sortable: true },
+        { label: 'Connected', key: 'connected', sortable: true },
+        { label: 'Last Seen', key: 'last_seen', sortable: true, useSortHandlerFn: true }
+      ],
+    }
+  },
+  methods: {
+    sortHandlerFn({ key, prevKey, sortColumnOrder, data}) {
+      return data.sort((a, b) => {
+        if (key === 'last_seen') {
+          if (sortColumnOrder === 'asc') {
+            if (a.last_ping > b.last_ping) {
```

Comment:
nit: I think you've got your conditional backwards. I'm pretty sure you want it ordered most recent (largest timestamp) to oldest (smallest timestamp) when you're sorting ascending, but looks like in the example you are doing oldest (smallest) date to most recent (largest)?

![image](https://user-images.githubusercontent.com/67973710/162979152-8aaeb930-f861-44b3-bbce-f2eda5f2f060.png)